drivers/interrupt_controller/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ zephyr_library_sources_ifdef(CONFIG_NXP_PINT                intc_nxp_pint.c)
 zephyr_library_sources_ifdef(CONFIG_RENESAS_RA_ICU          intc_renesas_ra_icu.c)
 zephyr_library_sources_ifdef(CONFIG_NXP_IRQSTEER            intc_nxp_irqsteer.c)
 zephyr_library_sources_ifdef(CONFIG_INTC_MTK_ADSP           intc_mtk_adsp.c)
+zephyr_library_sources_ifdef(CONFIG_WCH_PFIC                intc_pfic.c)
```

Comment:
Is  `pfic` manufacturer specific? If it is the case you need to add the manufacturer prefix on it.

```diff
-intc_pfic.c
+intc_wch_pfic.c
```

Reply:
Fixed now - thank you!